class StatusError(Exception):
    """Исключение статуса домашней роботы."""

    pass
